tests/functional/tests/base.py

```diff
@@ -24,8 +24,6 @@ def tearDown(self) -> None:
     def setUpClass(cls):
         """Run class setup for running tests."""
         super(BaseTestCase, cls).setUpClass()
-        cls.log_level = "info"
-        cli.config_logger(cls.log_level)
```

Comment:
Wouldn't it be better to set it to `debug` level instead?
(btw, I didn't find any usage of `cls.log_level` so we can remove it as a class variable)

Reply:
As we discussed it, no it's not.
Reasons:
1. We do not want to mix into user configuration define with zaza functest CLI (usage of `--log INFO`).
2. We do not want to use `DEBUG` mode as default, because novarc credential will be showed.